### DISCUSSION TITLE
No entendí la parte de kata 10

### DISCUSSION THREAD

**Discussion by AlanAguilarH** [LaunchX-InnovaccionVirtual/CursoIntroPython#335] [Q&A]
vi una parte pero me falla lo siguiente
A continuación, quita,ps el archivo config.txt y creamos un directorio denominado config.txt. Intentaremos llamar al archivo config.py para ver un error nuevo que debería ser similar al siguiente:

$ python config.py
Traceback (most recent call last):
  File "/tmp/config.py", line 9, in <module>
    main()
  File "/tmp/config.py", line 3, in main
    configuration = open('config.txt')
IsADirectoryError: [Errno 21] Is a directory: 'config.txt'

"quita,ps el archivo config.txt y creamos un directorio denominado config.txt." hay alguna idea como funciona

**Comment by GilbertoCanul** [comment]
¿Qué es lo que te aparece o te falla?

**Reply by TonaBau** [reply]
igual no se que es lo que se tiene que quitar

**Reply by TonaBau** [reply]
o que es ps

**Reply by TonaBau** [reply]
me sale error 13 en vez de error 21
![image](https://user-images.githubusercontent.com/99066214/154888924-997b90f3-d714-4fb3-b608-8510453f80b0.png)

**Reply by GilbertoCanul** [reply]
Creo que fue un error de dedo. Lo que hay que hacer es crear una carpeta que se llame "config.txt". También me apareció el "Permission denied" y he probado varias soluciones de internet, pero ninguna me ayuda.

**Comment by Canti21** [comment]
Segun entiendo el error IsADirectory solo aparece si utilizas la consola de linux.
En Windows si deberia aparecer Permission denied.
De hecho si lees la documentación aparece que la excepcion que debes comprobar es precisamente Permission denied.

![image](https://user-images.githubusercontent.com/81689973/154891673-460c9a46-be42-46cc-a520-94d4ab45ceff.png)

**Comment by JuanSalvador73** [comment]
Haz los ejercicios como se muestran en el documento en un archivo py o línea de comando - CMD -, si es este último saca screenshots. Yo hice ambas.
